src/services/celery_worker.py

```
@@ -1,6 +1,8 @@
 from celery import Celery
 import os
 from dotenv import load_dotenv
+from src.lib.crawl_service import CrawlService
```

Flake8 (7.2.0)

[error] 'src.lib.crawl_service.CrawlService' imported but unused

(F401)